Makefile

```diff
@@ -134,6 +134,8 @@ clean-sql-files:
 
 clean-extra:
 	@rm -f src/*~
+	@rm -f src/*.d
```

Comment:
These are already cleaned up by `make clean`. The reason you are seeing leftover files is because of renamed/removed source files when you do `git pull` and haven't cleaned before. Since those files are no long referenced in the Makefile they won't be cleaned either.

I guess this is safe (unless there are other .d and .o files that should not be removed), but generally not needed unless the above happens.